data/json/furniture_and_terrain/terrain-manufactured.json

```
@@ -1484,5 +1484,63 @@
       "sound_fail": "clang!",
       "items": [ { "item": "scrap", "count": [ 8, 32 ] } ]
     }
+  },
+  {
+    "type": "terrain",
+    "id": "t_brick_oven_base",
+    "name": "brick oven base",
+    "symbol": "=",
+    "description": "A flat assembly of carefully stacked bricks that is not quite an oven.",
+    "color": "yellow",
+    "move_cost": 4,
+    "deconstruct": { "ter_set": "t_dirt", "items": [ { "item": "fire_brick", "count": 30 } ] },
+    "bash": {
+      "str_min": 16,
+      "str_max": 25,
+      "sound": "whack!",
+      "sound_fail": "thunk.",
+      "ter_set": "t_dirt",
+      "items": [ { "item": "fire_brick", "count": [ 15, 20 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_brick_oven_struct",
+    "looks_like": "f_fireplace",
+    "name": "exposed brick oven",
+    "symbol": "#",
+    "description": "A large brick oven that could make several pizzas at once.  This one seems to be missing its door, and cannot be used in such a state.",
+    "color": "yellow",
+    "move_cost": 0,
+    "flags": [ "REDUCE_SCENT" ],
+    "deconstruct": { "ter_set": "t_dirt", "items": [ { "item": "fire_brick", "count": 90 } ] },
+    "bash": {
+      "str_min": 25,
+      "str_max": 50,
+      "sound": "whack!",
+      "sound_fail": "thunk.",
+      "ter_set": "t_dirt",
+      "items": [ { "item": "fire_brick", "count": [ 45, 75 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_brick_oven",
+    "looks_like": "f_fireplace",
+    "name": "brick oven",
+    "symbol": "#",
+    "description": "A large brick oven that could make several pizzas at once.  Smells faintly like pizza.",
```

```suggestion
    "description": "A large brick oven that could make several pizzas at once.",
```
Ince you can build it yourself I'm not sure the pizza smell makes sense

---

Fair enough. These sorts of large ovens can get some "seasoning" which leaves a residual smell (so I've been told) by means of fats/oils which melt and form a layer coating, much like the seasoning of cast iron cookware. But the player hasn't necessarily cooked anything which ends up smelling like pizza specifically.